modulesets-stable/gtk-osx-network.modules

```diff
@@ -60,8 +60,9 @@
 
   <autotools id="libtasn1" supports-non-srcdir-builds="no"
              autogen-sh="configure">
-    <branch repo="ftp.gnu.org" version="4.10"
-            module="libtasn1/libtasn1-4.10.tar.gz"/>
+    <branch repo="ftp.gnu.org" version="4.12"
+            module="libtasn1/libtasn1-4.12.tar.gz"
+            hash="sha256:6753da2e621257f33f5b051cc114d417e5206a0818fe0b1ecfd6153f70934753" />
   </autotools>
```

Comment:
libtasn1 is built from tarball in all three moduleset directories (modulesets-stable, modulesets, and modulesets-unstable) so it must be updated in all three.

Reply:
Will do.

Reply:
Done (2 commits as I used the github web ui):
https://github.com/totaam/gtk-osx-build/commit/ad820f039e3ee6f4242569a7ef4b3bd2250d8606
https://github.com/totaam/gtk-osx-build/commit/0e500ec23a6ac4dad8aa9da83ae1016ca36b9684